cli/tools/registry/pm.rs

```diff
@@ -593,6 +594,20 @@ enum PackageAndVersion {
   Selected(SelectedPackage),
 }
 
+fn is_exact_version_req(req: &PackageReq) -> bool {
+  match req.version_req.inner() {
+    RangeSetOrTag::RangeSet(range_set) => {
+      let items = &range_set.0;
+      if let Some(first) = items.first() {
+        first.start == first.end && items.len() == 1
```

Comment:
Oh, I just noticed the failing test and comment. I think you can check the `RangeBound`, but I'm seeing we accidentally don't export it from deno_semver: https://github.com/denoland/deno_semver/pull/37